src/region/__tests__/RegionAnnotations-test.tsx

```diff
@@ -81,6 +82,10 @@ describe('RegionAnnotations', () => {
             test('should reset the creator status', () => {
                 instance.handleStart();
 
+                expect(defaults.updateStaged).toHaveBeenCalledWith({
+                    location: defaults.page,
+                    shape: null,
+                });
                 expect(defaults.setStatus).toHaveBeenCalledWith(CreatorStatus.init);
```

Comment:
It seems like the status is now drifting from the original purpose. The value should be `null` if it's in the initial state.